### PR TITLE
Remove passwords

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description

This pull request:
* Makes the change of no longer storing admin Kerberos passwords on FreeIPA clients.  Instead, the IPA client configuration and the joining of the domain must be done manually by running a script.
* Removes the SystemD jazz since the client configuration and joining of the domain are no longer running automatically at boot.

## 💭 Motivation and Context

* We do not want to store sensitive passwords such as the Kerberos admin password on client machines.
* We made very similar changes in cisagov/ansible-role-freeipa-server#10.

## 🧪 Testing

I successfully deployed a new OpenVPN server to our staging environment with these changes.

## 🚥 Types of Changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by dav3r — APPROVED** [review]
Mean and clean!  🧹

**Review by hillaryj — APPROVED** [review]
👍

File: molecule/default/molecule.yml [hunk 1]
```diff
@@ -9,64 +9,24 @@ lint: |
...
-    image: geerlingguy/docker-ubuntu1804-ansible:latest
-    privileged: yes
-    volumes:
-      - /sys/fs/cgroup:/sys/fs/cgroup:ro
-    command: /lib/systemd/systemd
-    pre_build_image: yes
+  # - name: debian11
+  #   image: debian:bullseye-slim
+  # - name: kali
+  #   image: kalilinux/kali-rolling
```
Comment (felddy) [inline molecule/default/molecule.yml:21]: What's up with `debian11` and `kali`?

**Review by felddy — APPROVED** [review]
Looks great.  Big changes!  
See my question about the molecule platforms that are commented out.

File: molecule/default/molecule.yml (hunk 1 above)
Comment (jsf9k) [reply molecule/default/molecule.yml:21]: I'm not sure.  [Debian 11 (Bullseye, the current Debian testing) does not include the package right now, although Debian 12 (Sid, the current Debian unstable) does](https://packages.debian.org/search?keywords=freeipa).  Kali is always based on Debian testing, so it does not have the package either.

I assume this is an oversight that will be corrected prior to Debian Bullseye becoming the new Debian stable.

**Pull request merged**